src/metal.cs

```diff
@@ -405,11 +490,31 @@ public partial interface MTLDevice {
 		[NoTV]
 		[Export ("headless")]
 		bool Headless { [Bind ("isHeadless")] get; }
+		
+		[NoiOS, NoTV, NoWatch, Mac (10,12)]
+		[Abstract]
```

Comment:
likely right, but adding an `[Abstract]` is a breaking change - more in .NET than in ObjC but true in both (kind of work for ObjC as it's not enforced at runtime).